lib/devices.js

```diff
@@ -63,30 +60,40 @@ module.exports = function (log, db, push) {
   }
 
   function synthesizeName (device) {
-    var browserPart = part('uaBrowser')
-    var osPart = part('uaOS')
+    const uaBrowser = device.uaBrowser
+    const uaBrowserVersion = device.uaBrowserVersion
+    const uaOS = device.uaOS
+    const uaOSVersion = device.uaOSVersion
+    const uaFormFactor = device.uaFormFactor
```

Comment:
*Sigh*. I look forward to destructuring these to within an inch of their lives once we've ditched node 4.